tests/Common/SpanAssertion.php

```diff
@@ -366,4 +367,25 @@ public function isToBeSkipped()
     {
         return $this->toBeSkipped;
     }
+
+    /**
+     * Executes a callback only if the php version does not match one of the provided versions.
+     * Versions can be provided in the form: '5' -> all 5, '7.1' -> all 7.1.*, '7.1.2' -> only 7.1.2
+     * The callback will receive only one argument, which is the current assertion itself.
+     *
+     * @param string[] $versions
+     * @param Callable $callback
+     * @return $this
+     */
+    public function ifPhpVersionNotMatch(array $versions, $callback)
+    {
+        foreach ($versions as $version) {
+            if (Versions::phpVersionMatches($version)) {
+                return $this;
+            }
+        }
+
+        $callback($this);
+        return $this;
+    }
```

Comment:
The behavior here for more than one item is a bit fuzzy to me. Is there a reason for supporting anything other than a single element? So far you only have used it with exactly one item.

Otherwise, this PR looks good to me 👍

Reply:
good point. My original view was to also add `ifPhpVersionMatches` and use it in conditional assertion, where now we have to define the whole thing some place else and then do `if (php7 | 8) ...`. In this case I had anticipated we would have a few php7 & php 8 vs php5.

But we can always change later or find a different approach. Let me revert it back.